scripts/package.json

```diff
@@ -13,7 +13,6 @@
     "@microsoft/loader-load-themed-styles": "^1.6.0",
     "@uifabric/merge-styles": ">=5.13.0 <6.0.0",
     "autoprefixer": "^7.1.5",
-    "awesome-typescript-loader": "^3.2.3",
```

Comment:
Why the switch?

Reply:
i'm going to revert to trim down changes.